### PR TITLE
Added feature to check Linux distribution

### PR DESCRIPTION
**Pull request by d78ui98**
- [x] enabled distribution check
- [x] added help option

### CODE REVIEW

**Review by dashohoxha — COMMENTED** [review]
I think that it is easier to install sudo on Debian and then always use sudo on your scripts.

**Comment by d78ui98** [comment]
Agreed. But with this pull request the `test.sh` will run in all cases. Regardless of sudo installed in debian or not.

**Comment by dashohoxha** [comment]
You are right. But you can always install sudo in all the cases.

**Comment by d78ui98** [comment]
Okay. So should we close this one?

File: test.sh [hunk 1]
```diff
@@ -12,20 +13,35 @@ This script creates virtual network adapter which can be used for bridging
...
+fi
+
+if [[ "$DISTRO" == "Ubuntu" ]] || [[ "$DISTRO" == "LinuxMint" ]] ; then
+    use_sudo="sudo"
+fi
+
+if [[ "$use_sudo" == "" ]] && [[ "$UID" != 0 ]] ; then
+    echo "error: execute script as root user"
+    exit 1
+fi
```
Comment (dashohoxha) [inline test.sh:35]: You can check for the command `sudo` like this:
```bash
if [[ -z $(which sudo) ]]; then
    echo "Please install sudo first. For example see this tutorial:"
    echo "https://www.vultr.com/docs/how-to-use-sudo-on-debian-centos-and-freebsd"
    exit 1
fi
```

And that's all.

Comment (d78ui98) [reply test.sh:35]: Very nice!

**Pull request closed without merging**